### PR TITLE
Feature/iaco 1225 german translation of the swedish

### PR DESCRIPTION
**Pull request by moriaam**

### CODE REVIEW

**Comment by danevron** [comment]
👍

File: KlarnaOnDemand/KOD.bundle/de.lproj/localization.strings [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/* Klarna registration UINavigationController title */
+"REGISTRATION_TITLE" = "Zahlungsinformation";
+
+/* Klarna registration UINavigationController button */
```
Comment (TalyaS) [inline KlarnaOnDemand/KOD.bundle/de.lproj/localization.strings]: why are cancel and close in english?

**Comment by TalyaS** [comment]
LGTM

File: KlarnaOnDemand/KOD.bundle/de.lproj/localization.strings (hunk 1 above)
Comment (moriaam) [inline KlarnaOnDemand/KOD.bundle/de.lproj/localization.strings]: done

**Comment by tomerlevi444** [comment]
👍

**Pull request merged**